content/gocourse-2019-05-melbourne.slide

```diff
@@ -1003,6 +1003,43 @@ Use `gofmt` or `goimport` as part of your editor/IDE setup
 - ...
 
 
+* JSON - (De-)Serialisation
+
+- `Encode` converts Go struct, pointer or primitive data type into JSON encoded stream
+- `Marshal` converts Go struct, pointer or primitive data type into JSON byte array
+- `Decode` and `Unmarshall` are the inverse functions to `Encode` and `Marshal`
+
+	func Marshal(v interface{}) ([]byte, error) 
+
+	func Unmarshal(data []byte, v interface{}) error 
+
+
+* JSON - Tags
+
+- Attach `json` tag to specify JSON property name
```

Comment:
I suggest the this wording:

- Attach a 'json' field tag to customise (un)marshalling

I don't know what "specify JSON property name" means ("property" is not a JSON or Go word)

Reply:
Updated